control-plane/connect-inject/mesh_webhook.go

```diff
@@ -65,11 +71,8 @@ type MeshWebhook struct {
 	// TLSEnabled indicates whether we should use TLS for communicating to Consul.
 	TLSEnabled bool
 
-	// ConsulHTTPPort is the HTTP or HTTPs port we should use to talk to Consul.
-	ConsulHTTPPort string
-
 	// ConsulGRPCPort is the gRPC port we should use to talk to Consul.
-	ConsulGRPCPort string
+	ConsulGRPCPort int
```

Comment:
not blocking: while it isnt directly required by the Consul config, given we are moving ConsulHTTPPort into the ConsulConfig, it feels like a symmetric change to also move ConsulGRPCPort into Consul Config.

Reply:
yes, good point!

Reply:
done!